Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Merchant.hs

```diff
@@ -643,11 +647,48 @@ data CreateMerchantOperatingCityReq = CreateMerchantOperatingCityReq
   deriving stock (Eq, Show, Generic)
   deriving anyclass (ToJSON, FromJSON, ToSchema)
 
-instance HideSecrets CreateMerchantOperatingCityReq where
-  hideSecrets = identity
+instance FromMultipart Tmp CreateMerchantOperatingCityReq where
+  fromMultipart form = do
+    CreateMerchantOperatingCityReq
+      <$> fmap fdPayload (lookupFile "file" form)
+      <*> fmap fdFileCType (lookupFile "file" form)
+      <*> fmap (read . T.unpack) (lookupInput "city" form)
+      <*> fmap (read . T.unpack) (lookupInput "state" form)
+      <*> fmap (read . T.unpack) (lookupInput "lat" form)
+      <*> fmap (read . T.unpack) (lookupInput "long" form)
+      <*> fmap (read . T.unpack) (lookupInput "primaryLanguage" form)
+      <*> fmap (read . T.unpack) (lookupInput "supportNumber" form)
+      <*> fmap (read . T.unpack) (lookupInput "enableForMerchant" form)
+      <*> fmap (read . T.unpack) (lookupInput "exophone" form)
+      <*> fmap (read . T.unpack) (lookupInput "rcNumberPrefixList" form)
 
 newtype CreateMerchantOperatingCityRes = CreateMerchantOperatingCityRes
   { cityId :: Text
   }
   deriving stock (Eq, Show, Generic)
   deriving anyclass (ToJSON, FromJSON, ToSchema)
+
+type CreateMerchantOperatingCityAPIT =
```

Comment:
Please remove the other `CreateMerchantOperatingCityAPI` and `CreateMerchantOperatingCityReq` as they are not required anymore.